src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractViolationReporter.java

```diff
@@ -143,8 +143,8 @@ protected void finishLocalSetup() throws CheckstyleException {
     /**
      * Log a message that has no column information.
      *
-     * @param line the line number where the error was found
-     * @param key the message that describes the error
+     * @param line the line number where the audit event was found
+     * @param key the message that describes the audit events
```

Comment:
This line should be singular. This is the message that describes the singular audit event.

Reply:
fixed.